src/panels/config/automation/add-automation-element-dialog.ts

```diff
@@ -318,17 +317,15 @@ class DialogAddAutomationElement extends LitElement implements HassDialog {
             (!domainUsed && manifest?.integration_type === "entity") ||
             !["helper", "entity"].includes(manifest?.integration_type || "")))
       ) {
-        const icon = domainIconWithoutDefault(domain);
         result.push({
           group: true,
-          iconPath: icon,
-          image: !icon
```

Comment:
Why was brand image removed?

Reply:
The logic has been moved into ha-domain-icon.